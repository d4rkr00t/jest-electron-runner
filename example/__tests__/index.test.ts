import * as add from "../index";

describe("Some test", () => {
  test("should add 1 + 2", () => {
    expect(add(1, 2)).toBe(3);
  });

  test("should add 1 + 1", () => {
    expect(add(1, 1)).toBe(3);
  });

  test("test with snapshot", () => {
    expect({ a: add(1, 2), b: add(3, 4) }).toMatchSnapshot();
  });
});
