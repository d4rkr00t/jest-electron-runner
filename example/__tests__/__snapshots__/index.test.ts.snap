// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Some test test with snapshot 1`] = `
Object {
  "a": 3,
  "b": 7,
}
`;
